package.json

```diff
@@ -169,7 +169,17 @@
         ],
         "url": "./manifest-schema.json"
       }
-    ]
+    ],
+    "configuration": {
+      "title": "flatpak-vscode",
+      "properties": {
+        "flatpak-vscode.enableRustAnalyzerServerPathOverride": {
```

Comment:
I don't think this should be specific to rust, but we should allow the user to disable any kind of "integration". I would rename it to something generic

Reply:
Any thoughts how to name and describe it exactly? Not a native speaker and a hobbyist, so I tend to have trouble finding the precise wording for things like this

Reply:
I would name it `extensionsIntegration`, for the description you can put something like: `integrate flatpak-vscode with other Flatpak extensions like rust-analyzer. Enabling this will make the integrated extensions run their binaries inside the Flatpak sandbox instead of running on the host.`